resotocore/resotocore/core_config.py

```diff
@@ -62,6 +62,28 @@ def inside_docker() -> bool:
     )
 
 
+def inside_kubernetes() -> bool:
+    """
+    Try to detect if we are running on kubernetes.
+    """
+    return (
+        # environment variables have to be set explicitly
+        os.environ.get("HELM_VERSION", "false") != "false"
+        or any((True for x in os.environ if x.startswith("KUBERNETES_")))
+    )
+
+
+def helm_installation() -> bool:
+    """
+    Try to detect if we were installed via helm chart.
+    """
+    return (
+        # environment variables have to be set explicitly
+        os.environ.get("HELM_VERSION", "false")
+        != "false"
```

Comment:
```suggestion
        "HELM_VERSION" in os.environ
```
Not wrong, but maybe this one is easier to read?